### DISCUSSION TITLE
Implementing Inventree for Apparel with different sizes and combos

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#4870] [Q&A]
I have a part **Shirt**. A **Shirt** is made of **Fabric, Buttons, Label, Tags and others(BOM)**. The **Shirt** is then **Manufactured(Build)** in different sizes **S,M,L,XL etc**. 
So I have created a part **Shirt** with different variation **S,M,L,XL** and having the above BOM. 

While creating the build How do I  specify that the output of the build will be the variation **S,M,L,XL** and not to the part **Shirt** ?
The part **Shirt** should add up the variation and show the total quantity.

**Comment by SchrodingersGat** [comment]
@rohitsingal1 suggested approach would be to make the "Shirt" a template part, with "variants" for S, M, L, XL etc.

Each variant could either share the same BOM as the parent template part, or have a slightly different BOM if required.

The *output* of a build order must match the part referenced by the build
